l10n/en/mozorg/home-m24.ftl

```diff
@@ -69,6 +69,11 @@ m24-home-mozilla-builders-v2= { -brand-name-mozilla-builders }
 m24-home-builders-helps-independent = Builders helps independent developers create transformative open-source AI projects through bespoke collaborations, programming, and community.
 m24-home-builders-helps-independent-v2 = { -brand-name-mozilla-builders } helps independent developers create transformative open-source AI projects through bespoke collaborations, programming, and community.
 # Used as an accessible text alternative for an image
+m24-home-alt-mozfest = A large group of people gathered in a busy indoor event space, viewed from above. Attendees are standing, talking, and looking at electronic devices around tables with laptops and equipment. The scene is framed by a bright yellow graphic border.
```

Comment:
alt translation is not required for block to show, only the title, body, and CTA text are checked